tests/Iterator/MessageManagerMessageIteratorTest.php

```diff
@@ -15,6 +15,9 @@
 
 use Doctrine\Common\Persistence\ManagerRegistry;
 use PHPUnit\Framework\TestCase;
+use Sonata\NotificationBundle\Iterator\MessageManagerMessageIterator as MessageManagerMessageIteratorObject;
```

Comment:
why the `as` keyword?

Reply:
because of existence of Sonata\NotificationBundle\Tests\Iterator\MessageManagerMessageIterator class. But to test order, i need to mock MessageManager::findByTypes